### ISSUE TITLE
attractors to dataframe fails when networks > 32 nodes

### ISSUE THREAD

**Issue by ArturoArciniega** [mar-esther23/boolnet-perturb#6]
When attempting to transform the attractors into a dataframe for a network containing more than 32 nodes, the **attractorsTodataframe** function does not take into account binary representations exceeding 31 bits due to a limitation in R, which only allows bit operations up to 31 bits. As a result, it is not possible to completely represent the attractors in a dataframe due to the presence of missing data.

**Comment by mar-esther23** [comment]
Yep. The problem is in the export to dataframe function and is related with how boolnet perturb represents large networks. Sincerly, I don't know if I'll fix it, but if someone has the time I can incorporate the code and solve.
